Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:latest
+
+WORKDIR /build
+
+# Copy only install-dependencies.sh 
+# This helps with layer caching
+COPY ci-tests/install-dependencies.sh /build
+
+# Install dependencies
+RUN apt-get update && apt-get install -y \
+    sudo wget python3 python3-pip python-is-python3
```

Comment:
I am surprised that none of these come in the ubuntu base image. Is it that bare-bones? Note that wget is installed by install-dependencies.sh. Why do we need python3-pip and python-is-python3?

Reply:
OK, just ran the ubuntu:latest image, and I see it is really barebones... I would still remove python-is-python3.